mlrun/package/packagers/python_standard_library_packagers.py

```diff
@@ -0,0 +1,616 @@
+            file_format = StructFileSupportedFormat.match_format(path=file_path)
+        if file_format is None:
+            raise MLRunInvalidArgumentError(
+                f"File format of {data_item.key} ('{''.join(pathlib.Path(file_path).suffixes)}') is not supported. "
+                f"Supported formats are: {' '.join(StructFileSupportedFormat.get_all_formats())}"
+            )
+
+        # Read the object:
+        formatter = StructFileSupportedFormat.get_format_handler(fmt=file_format)
+        obj = formatter.read(file_path=file_path)
+
+        return obj
+
+
+class DictPackager(_BuiltinCollectionPackager):
+    """
+    ``builtins.dict`` packager.
+    """
+
+    PACKABLE_OBJECT_TYPE = dict
+
+    @classmethod
+    def unpack_file(cls, data_item: DataItem, file_format: str = None) -> dict:
+        """
+        Unpack a dictionary from file.
+
+        :param data_item:   The data item to unpack.
+        :param file_format: The file format to use for reading the dictionary. Default is None - will be read by the
+                            file extension.
+
+        :return: The unpacked dictionary.
+        """
+        # Unpack the object:
+        obj = super().unpack_file(data_item=data_item, file_format=file_format)
+
+        # Check if needed to cast from list:
+        if isinstance(obj, list):
+            return {index: element for index, element in enumerate(obj)}
+        return obj
+
+
+class ListPackager(_BuiltinCollectionPackager):
+    """
+    ``builtins.list`` packager.
+    """
+
+    PACKABLE_OBJECT_TYPE = list
+
+    @classmethod
+    def unpack_file(cls, data_item: DataItem, file_format: str = None) -> list:
+        """
+        Unpack a list from file.
+
+        :param data_item:   The data item to unpack.
+        :param file_format: The file format to use for reading the list. Default is None - will be read by the file
+                            extension.
+
+        :return: The unpacked list.
+        """
+        # Unpack the object:
+        obj = super().unpack_file(data_item=data_item, file_format=file_format)
+
+        # Check if needed to cast from dict:
+        if isinstance(obj, dict):
+            return list(obj.values())
+        return obj
+
+
+class TuplePackager(ListPackager):
+    """
+    ``builtins.tuple`` packager.
+
+    Notice: a ``tuple`` returned from a function is usually treated as multiple returned objects, and so MLRun will try
+    to pack each of them separately and not as a single tuple. For example::
+
+        def example_func_1():
+            return 10, [1, 2, 3], "Hello MLRun"
+
+    Will be returned as a ``tuple`` of 3 items: `(10, [1, 2, 3], "Hello MLRun")` but the items will be packaged
+    separately one by one and not as a single ``tuple``.
+
+    In order to pack tuples (not recommended), use the configuration::
+
+        mlrun.mlconf.packagers.pack_tuple = True
+
+    Or more correctly, cast your returned tuple to a ``list`` like so::
```

Comment:
I would recommend this way more than using the one above, for reasons I mentioned in other comments...
Could also do `return [my_tuple]`.

Reply:
again, its for extreme user who really want to log tuples for some reason... I can remove it in a minute.